packages/documentation/copy/en/reference/Type Compatibility.md

```diff
@@ -288,3 +288,30 @@ These differ only in that assignment extends subtype compatibility with rules to
 
 Different places in the language use one of the two compatibility mechanisms, depending on the situation.
 For practical purposes, type compatibility is dictated by assignment compatibility, even in the cases of the `implements` and `extends` clauses.
+
+## Any, unknown, object, void, undefined, null, and never
```

Comment:
I thought about this a bit, and I think this is probably the right call. Primitives isn't a great option either, basics is a bit too ambiguous and I think this is the only real improvement I can consider, but I'm fine with it either way:

```suggestion
## `Any`, `unknown`, `object`, `void`, `undefined`, `null`, and `never` assignability
```